### PR TITLE
Support MP4 video backgrounds

### PR DESCRIPTION
**Pull request by reco1I**

### CODE REVIEW

File: src/com/edlplan/ui/fragment/InGameSettingMenu.java [hunk 1]
```diff
@@ -116,6 +117,15 @@ private void reload() {
                     .commit();
         });
 
+        enableVideo = findViewById(R.id.enableVideo);
+        enableVideo.setChecked(Config.isEnableVideo());
+        enableVideo.setOnCheckedChangeListener((buttonView, isChecked) -> {
+            Config.setEnableVideo(isChecked);
```
Comment (Beyley) [inline src/com/edlplan/ui/fragment/InGameSettingMenu.java:123]: I feel like this should be a NOP if the OS version is too low *here*, and maybe with a message saying "Your android version is too old to play video!"

File: res/values/options.xml [hunk 2]
```diff
@@ -337,4 +337,7 @@
 
     <string name="opt_display_realtime_pp_counter_title">Display real-time PP counter</string>
     <string name="opt_display_realtime_pp_counter_summary">Displays a real-time PP counter during gameplay. May increase loading time significantly, especially in long beatmaps</string>
+
+    <string name="opt_enable_video_title">Enable video</string>
+    <string name="opt_enable_video_summary">Enable video playback during gameplay (Android 6 or newer)</string>
```
Comment (Rian8337) [inline res/values/options.xml:342]: If we're going to support non-1x speed multiplier video playback in devices below Android 6, this description is inaccurate.

File: src/ru/nsu/ccfit/zuev/osu/beatmap/sections/BeatmapEvents.java [hunk 3]
```diff
@@ -14,6 +14,16 @@ public class BeatmapEvents {
...
+    /**
+     * The file name of this beatmap's background video.
+     */
+    public String videoFilename;
+
+    /**
+     * The beatmap's background video start time.
+     */
+    public int videoStartTime;
+
```
Comment (Rian8337) [inline src/ru/nsu/ccfit/zuev/osu/beatmap/sections/BeatmapEvents.java:26]: These need to be inserted to the copy constructor as well.

**Comment by reco1I** [comment]
Will be reworked on top of #257 branch.

**Pull request closed without merging**